tests/queries/test_bulk_update.py

```diff
@@ -200,6 +200,13 @@ def test_field_references(self):
         Number.objects.bulk_update(numbers, ['num'])
         self.assertCountEqual(Number.objects.filter(num=1), numbers)
 
+    def test_row_count(self):
+        numbers = [Number.objects.create(num=0) for _ in range(10)]
+        for number in numbers:
+            number.num = F('num') + 1
+        rows = Number.objects.bulk_update(numbers, ['num'])
+        self.assertEqual(rows, 10)
+
```

Comment:
I don't think we need this new test and should update existing tests instead.

For example, you could check that `0` is returned in `test_empty_objects()` and that `2000` is returned in `test_large_batch()`. I imagine these would be sufficient.